config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :state_file_archived_intake_requests
```

Comment:
[dust] why is this at the top of the file, can it be closer to the routes that are relevant?